tequilapi/endpoints/proposals.go

```diff
@@ -0,0 +1,38 @@
+package endpoints
+
+import (
+	"github.com/julienschmidt/httprouter"
+	"github.com/mysterium/node/server"
+	dto_discovery "github.com/mysterium/node/service_discovery/dto"
+	"github.com/mysterium/node/tequilapi/utils"
+	"net/http"
+)
+
+type proposalsEndpoint struct {
+	mysteriumClient server.Client
+}
+
+type proposalsDto struct {
+	Proposals []dto_discovery.ServiceProposal `json:"proposals"`
+}
```

Comment:
This struct is duplicate with `server.dto.ProposalsResponse`, maybe we can keep only one of those?

Reply:
your are right, but as of Waldz reqiurement, we want to decouple tequila endpoint structures from internals of app and from discovery.

Reply:
Two different apis should use their own internal serialization/deserialization structures as these are absolutely independent ones - no matter that data is the same at the moment. Agree with @shroomist

Reply:
Totally agree - I had this idea in mind, just wanted it to be clarified :D